common/src/commonMain/kotlin/com/trikot/sample/viewmodels/base/AbstractViewModelController.kt

```diff
@@ -0,0 +1,18 @@
+package com.trikot.sample.viewmodels.base
+
+import com.mirego.trikot.foundation.ref.weakAtomicReference
+import com.mirego.trikot.streams.cancellable.CancellableManager
+import com.mirego.trikot.viewmodels.ViewModel
+
+abstract class AbstractViewModelController<NavigationDelegate : BaseNavigationDelegate, VM : ViewModel> : ViewModelController() {
```

Comment:
BaseViewModelController maybe?
I dont think "abstract" should be in a class name. It doesn't mean anything regarding what this object is all about.

Reply:
+1 Especially since it is already an `abstract class`

Reply:
I don't agree with you, but since I don't want to burn calories here ... I'll change it.